dockers/docker-snmp/snmpd.conf.j2

```diff
@@ -34,16 +34,56 @@ view   systemonly  included   .1.3.6.1.2.1.1
 view   systemonly  included   .1.3.6.1.2.1.25.1
 
                                                  #  Default access to basic system info
-{% if snmp_rocommunities %}
+{% if SNMP_COMMUNITY is defined %}
+
+{% for community in SNMP_COMMUNITY %}
+{% if SNMP_COMMUNITY[community]['TYPE'] == 'RO' %}
+rocommunity {{ community }}
+rocommunity6 {{ community }}
+{% elif SNMP_COMMUNITY[community]['TYPE'] == 'RW' %}
+rwcommunity {{ community }}
+rwcommunity6 {{ community }}
+{% endif %}
+{% endfor %}
+{% endif %}
+
+{% if SNMP_COMMUNITY is defined and snmp_rocommunities is defined %}
 {% for community in snmp_rocommunities %}
+{% if community not in SNMP_COMMUNITY %}
 rocommunity {{ community }}
 rocommunity6 {{ community }}
+{% endif %}
 {% endfor %}
-{% else %}
+{% elif community not in SNMP_COMMUNITY %}
```

Comment:
This condition check is the same as condition in line 52.

Reply:
probably this block and below block can be combined as :
{% if snmp_rocommunities is defined %}
{% for community in snmp_rocommunities %}
{% (if SNMP_COMMUNITY is defined and community not in SNMP_COMMUNITY) or (SNMP_COMMUNITY is not defined) %}
rocommunity {{ community }}	rocommunity {{ community }}
rocommunity6 {{ community }}	rocommunity6 {{ community }}
{% endif %}	
{% endfor %}	
{% endif %}

Reply:
Updated the Jinja file to only look at the configdb as the snmp.yml info was imported into the configdb via the snmp_yml_to_configdb.py that is run each time the container is restarted. (via the start.sh)

Reply:
That original comment is no longer necessary with these changes.